projects/control-service/projects/helm_charts/pipelines-control-service/templates/alertmanager.yaml

```diff
@@ -3,34 +3,29 @@
   SPDX-License-Identifier: Apache-2.0
  */}}
 
-{{ - /*
-      Copyright 2021-2023 VMware, Inc.
-    SPDX-License-Identifier: Apache-2.0
-      */ }}
-
-{{ - if .Values.alertmanager.enabled }}
+{{- if .Values.alertmanager.enabled }}
 apiVersion: monitoring.coreos.com/v1alpha1
 kind: AlertmanagerConfig
-  labels: {{ - include "pipelines-control-service.labels" . | nindent 4 }}
-    {{ - range $key, $value := .Values.alertmanager.labels }}
+  labels: {{- include "pipelines-control-service.labels" . | nindent 4 }}
+    {{- range $key, $value := .Values.alertmanager.labels }}
     {{ $key }}: {{ $value | quote }}
-    {{ - end }}
+    {{- end }}
   name: {{ .Values.alertmanager.name }}
   namespace: {{ .Values.alertmanager.namespace }}
   spec:
     receivers:
       route: {{ toYaml .Values.alertmanager.spec.receivers.route | nindent 8 }}
     emailConfigs: {{ toYaml .Values.alertmanager.spec.emailConfigs | nindent 6 }}
-      authPassword: {{ - with secret .Values.alertmanager.authentication.secretName - }}
+      authPassword: {{` {{- with secret .Values.alertmanager.authentication.secretName -}}
         {{ .Data.authPassword | b64dec }}
-        {{ - end }}
-      authSecret: {{ - with secret .Values.alertmanager.authentication.secretName - }}
+        {{- end }} `}}
+      authSecret: {{` {{- with secret .Values.alertmanager.authentication.secretName -}}
         {{ .Data.authSecret | b64dec }}
-        {{ - end }}
-      authSecretKey: {{ - with secret .Values.alertmanager.authentication.secretName - }}
+        {{- end }} `}}
+      authSecretKey: {{` {{- with secret .Values.alertmanager.authentication.secretName -}}
         {{ .Data.authSecretKey | b64dec }}
-        {{ - end }}
-      authIdentity: {{ - with secret .Values.alertmanager.authentication.secretName - }}
+        {{- end }} `}}
+      authIdentity: {{` {{ - with secret .Values.alertmanager.authentication.secretName -}}
         {{ .Data.authIdentity | b64dec }}
-        {{ - end }}
-  {{ - end }}
+        {{- end }} `}}
```

Comment:
what is the backtick needed for ?

Reply:
removed in favor of https://docs.openshift.com/container-platform/4.10/rest_api/monitoring_apis/alertmanagerconfig-monitoring-coreos-com-v1alpha1.html#spec-receivers-emailconfigs-authpassword which is more concise